pinecone/__version__

```diff
@@ -1 +1 @@
-2.2.0
```

Comment:
A past error meant this version number was out of sink with what has already been published to pypi. So needed to update that for future releases.